lib/rules/aria-empty-table-header.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "aria-empty-table-header",
```

Comment:
Since this rule works on table elements and is not validating aria roles / attributes, it would probably be better to drop the `aria` from the name and just call the rule `empty-table-header`